<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fermady Prices</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagenes/hucha.png</normaloff>imagenes/hucha.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color:qconicalgradient(cx:1, cy:0.648, angle:0, stop:0 rgba(169, 178, 207, 255), stop:0.596591 rgba(220, 209, 218, 255), stop:0.98 rgba(66, 66, 66, 255), stop:1 rgba(0, 0, 0, 0));</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>60</y>
              <width>359</width>
              <height>32</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Leelawadee UI</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTipDuration">
             <number>0</number>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#f7f7f7;&quot;&gt;¡EMPIEZA A HACER TRACKING!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="btnos">
            <property name="geometry">
             <rect>
              <x>680</x>
              <y>-10</y>
              <width>131</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>8</pointsize>
              <underline>true</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
        border: none; 

	color: rgb(213, 213, 213);
        padding: 10px 20px;
}</string>
            </property>
            <property name="text">
             <string>Sobre nosotros</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(222, 222, 222);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: transparent;
}
QPushButton:hover{
 	background-color: rgb(199, 199, 199);
 	border-radius:10px; 
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <property name="bottomMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnGallo">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Amplia gama de productos, lo último en tecnología de las marcas más exclusivas del país &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>-2</number>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imagenes/GMG.png</normaloff>imagenes/GMG.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>179</width>
                   <height>143</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnSycom">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tienda de accesorios para computadoras en Tegucigalpa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>-2</number>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imagenes/Sycom.png</normaloff>imagenes/Sycom.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>179</width>
                   <height>173</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnJets">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tecnología e innovación a tu medida con Jetstereo!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>-2</number>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imagenes/Jetstereo.png</normaloff>imagenes/Jetstereo.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>169</width>
                   <height>173</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
              <zorder>btnGallo</zorder>
              <zorder>btnJets</zorder>
              <zorder>btnSycom</zorder>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: transparent;
}
QPushButton:hover{
 	background-color: rgb(199, 199, 199);
 	border-radius:10px; 
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <property name="bottomMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnTecknos">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>8</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diunsa |La tienda departamental más grande de Honduras. Compra en línea lo mejor de electrónica, electrodomésticos, tecnología, hogar, deporte y mucho más.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>-2</number>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imagenes/imgp/diun.png</normaloff>imagenes/imgp/diun.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>215</width>
                   <height>173</height>
                  </size>
                 </property>
                 <property name="autoDefault">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnLadyLee">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Todo para mi Hogar! La tienda por departamentos más grande y completa de Honduras.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>-2</number>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imagenes/LadyLee.png</normaloff>imagenes/LadyLee.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>221</width>
                   <height>180</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRadioshack">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;RadioShack Corporation es una empresa estadounidense, que gestiona una cadena de tiendas de artículos y componentes electrónicos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="toolTipDuration">
                  <number>-2</number>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>imagenes/Radioshack.png</normaloff>imagenes/Radioshack.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>179</width>
                   <height>173</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
